src/vmm/src/devices/legacy/i8042.rs

```diff
@@ -140,6 +140,7 @@ impl I8042Device {
         }
     }
 
+    /// Signal a ctrl-alt-del (reset) event.
```

Comment:
This seems wrong, the ctrl+alt+del one is `trigger_ctrl_alt_del` a few lines above. This one just seems to write an arbitrary byte into the buffer?

Reply:
removed comment